core.py

```diff
@@ -9,8 +9,19 @@
 
 import os
 import sys
-from ConfigParser import ConfigParser
-from StringIO import StringIO
+import platform
+
+try:
+	# try to import py2 modules
+	from ConfigParser import ConfigParser
+	from StringIO import StringIO
+	PY3 = False
+except ImportError:
+	# py 3
+	from configparser import ConfigParser
+	from io import StringIO, IOBase as file
```

Comment:
The `io` module exists under Python 2 as well and is basically identical. It might be better to always use the types from `io` instead of the legacy Python 2 names.